src/main/java/com/emeq/counter/model/Food.java

```diff
@@ -0,0 +1,29 @@
+package com.emeq.counter.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+
+import com.emeq.counter.dto.FoodDTO;
```

Comment:
as a rule of thumb, can we always keep project imports after library imports?

Reply:
I organized all imports.